code/game/say.dm

```diff
@@ -50,8 +50,9 @@ GLOBAL_LIST_INIT(freqtospan, list(
 	//Start name span.
 	var/spanpart2 = "<span class='name'>"
 	//Radio freq/name display
-	var/job = speaker.GetJob()
-	var/freqpart = radio_freq ? "\[[get_radio_name(radio_freq)][job ? " ([job])": ""]\] " : ""
+	var/freqpart = compose_freq(speaker, radio_freq)
+	//Job display
+	var/jobpart = compose_job(speaker, message_language, raw_message, radio_freq)
```

Comment:
Why not defines?

Reply:
Give an example and I'll implement it.

Reply:
```
#define CMSG_FREQPART compose_freq(speaker, radio_freq)
#define CMSG_JOBPART compose_job(speaker, message_language, raw_message, radio_freq)
/atom/movable/proc/compose_message(atom/movable/speaker, datum/language/message_language, raw_message, radio_freq, list/spans, message_mode, face_name = FALSE)
(...)
return "[spanpart1][spanpart2][CMSG_FREQPART][languageicon][CMSG_JOBPART][namepart][endspanpart][messagepart]"
(...)
#undef CMSG_FREQPART
#undef CMSG_JOBPART
```

Reply:
I don't really see it as being cleaner, but if you really want it, I can do it